lock-api/src/main/java/com/palantir/lock/v2/TimelockRpcClient.java

```diff
@@ -57,27 +55,19 @@
     LockImmutableTimestampResponse lockImmutableTimestamp(IdentifiedTimeLockRequest request);
 
     @POST
-    @Path("start-atlasdb-transaction")
-    StartAtlasDbTransactionResponse deprecatedStartAtlasDbTransaction(IdentifiedTimeLockRequest request);
-
-    @POST
-    @Path("start-identified-atlasdb-transaction")
-    StartIdentifiedAtlasDbTransactionResponse deprecatedStartIdentifiedAtlasDbTransaction(
+    @Path("start-atlasdb-transaction-v3")
+    StartAtlasDbTransactionResponseV3 deprecatedStartTransaction(
             StartIdentifiedAtlasDbTransactionRequest request);
```

Comment:
removing deprecated methods from rpcClient while keeping endpoints on the server (planing to remove those also with 1.x release)